rustic-cargo.el

```diff
@@ -349,6 +351,53 @@ Execute process in PATH."
                                        'rustic-cargo-outdated-upgrade)))))
       (tabulated-list-put-tag "U" t))))
 
+;;;###autoload
+(defun rustic-cargo-mark-latest-upgrade ()
+  "Mark an upgradable package to the latest available version."
+  (interactive)
+  (let* ((crate (tabulated-list-get-entry (point)))
+         (v (substring-no-properties (elt crate 3)))
+         (inhibit-read-only t))
+    (when v
+      (save-excursion
+        (goto-char (line-beginning-position))
+        (save-match-data
+          (when (search-forward (elt crate 0))
+            (replace-match (propertize (elt crate 0)
+                                       'font-lock-face
+                                       'rustic-cargo-outdated-upgrade)))
```

Comment:
Maybe use a `lambda` ?

Reply:
Sorry, not sure I understand what to do here. Can you probably provide a sample code ? Thanks!

Reply:
I mean you could put the call to `replace-match` into a lambda that is in the let bindings like in https://github.com/brotzeit/rustic/blob/01c3b34016656d7dcf36511b9f6410d71a112e29/rustic-racer.el#L325

Reply:
Thanks, updated.